Tests/Builder/FormContractorTest.php

```diff
@@ -46,4 +47,68 @@ public function testGetFormBuilder()
             $this->formContractor->getFormBuilder('test', array('foo' => 'bar'))
         );
     }
+
+    public function testDefaultOptionsForSonataFormTypes()
+    {
+        $admin = $this->getMockBuilder('Sonata\AdminBundle\Admin\AdminInterface')->getMock();
```

Comment:
I think you can do this directly : `$this->getMock('Sonata\AdminBundle\Admin\AdminInterface');`